experimental/framework/tinylicious-client/src/interfaces.ts

```diff
@@ -31,6 +31,11 @@ export interface TinyliciousContainerServices {
     audience: ITinyliciousAudience;
 }
 
+export interface TinyliciousContainerAndServices {
+    fluidContainer: FluidContainer;
+    containerServices?: TinyliciousContainerServices;
```

Comment:
Do we have a scenario where we return a `fluidContainer` but no `containerServices`?  If not, we might want to have both of these be defined in type declaration

Reply:
I believe we don't. I will define those in type declaration